pkg/kubectl/cmd/wait/wait.go

```diff
@@ -105,11 +105,11 @@ func NewCmdWait(restClientGetter genericclioptions.RESTClientGetter, streams gen
 	flags := NewWaitFlags(restClientGetter, streams)
 
 	cmd := &cobra.Command{
-		Use:                   "wait resource.group/name [--for=delete|--for condition=available]",
+		Use: "wait resource.group/name [--for=delete|--for condition=available]",
```

Comment:
gofmt does not look right for this. Compare with https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/edit/edit.go#L75 for example